cdc/sink/mysql/mysql.go

```diff
@@ -533,7 +524,7 @@ func (s *mysqlSink) cleanTableResource(tableID model.TableID) {
 
 func (s *mysqlSink) Close(ctx context.Context) error {
 	s.execWaitNotifier.Close()
-	s.resolvedNotifier.Close()
+	close(s.resolvedCh)
```

Comment:
Why is it necessary to close `s.resolvedCh`?
Can `Close(...)` race with `FlushRowChangedEvents(...)`? If so, `FlushRowChangedEvents` will panic if it tries to send to a closed channel.

Reply:
Yes, there are data race problem, thank you for your review